test/very_good_infinite_list_test.dart

```diff
@@ -313,6 +313,52 @@ void main() {
         },
       );
     });
+
+    group('goldens', () {
+      const tags = 'golden';
+      String goldenPath(String fileName) => 'goldens/$fileName.png';
+
+      testWidgets(
+        'renders successfully when scrolled vertically to bottom',
+        tags: tags,
```

Comment:
Any reason why this is tagged? Can't they just run in parallel with the others?

Reply:
Used to generate golden files `flutter test -t golden --update-goldens` without running all tests.